election/serializers.py

```diff
@@ -83,15 +83,28 @@ class Meta:
 
 # See https://github.com/MieuxVoter/mvapi/pull/5#discussion_r291891403 for explanations
 class Candidate:
-    def __init__(self, name, idx, profile, grade):
+    def __init__(self, name, idx, profile, grade, score, num_votes):

```

Comment:
I deleted score and num_votes, because num_votes is already contained in profile. And score is pretty much useless (very easy to recover from profile, not clear who is going to use it)